### PR TITLE
Feature/1 app basic structure setup

### LINKED ISSUE #1: app basic structure setup
setup Github settings | 1h
edit readMe | 1h
crate logo and title logo | 1h
add Components | 1h
add Pages | 1h
setup React Router | 1h
add Library's in list | 1h
setup basic CSS design | 4h

### PR DESCRIPTION
**Pull request by AndrewShmorhunGit**
Hollo ✌ Check this out) I need to do few more logos, and may be change structure in the components, but may be you see something more? Good morning my senior 😎 have a nice day)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,35 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+- Maybe this section is not needed.
```
Comment (silvertipbear) [inline README.md:41]: This section is needed and most important.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,35 @@
...
-## Available Scripts
+```
+Demo version of famous delivery application 'Glovo'
+```
 
-In the project directory, you can run:
+App presentation using development best practices like:
 
-### `npm start`
+1. React & TypeScrypt
```
Comment (silvertipbear) [inline README.md]: Always check for typos. Use spellchecking plugin for VS Code.

File: README.md [hunk 3]
```diff
@@ -1,46 +1,35 @@
...
+2. Redux, redux-tool-kit
+3. Git and Github
+4. Deployed by netlify
```
Comment (silvertipbear) [inline README.md]: Is it deployed to Netlify right now? Put only valid information here.

File: src/components/FastServices.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+const FastServices = () => {
+  return <h1>FastServices</h1>;
+};
+
+const Wrapper = styled.main`
```
Comment (silvertipbear) [inline src/components/FastServices.tsx:7]: You have a lot of unused variables in your code. Use them, export them or delete them. Also, add noUnusedLocals option to the compiler settings.

**Review by silvertipbear — CHANGES_REQUESTED** [review]
I don't see implemented header and footer. Also, header or footer should contain a primitive navigation bar that can be used to test router functionality.

File: README.md (hunk 1 above)
Comment (AndrewShmorhunGit) [reply README.md:41]: Ok. '- Maybe this section is not needed.' - was about coming after all previous info.

File: README.md (hunk 3 above)
Comment (AndrewShmorhunGit) [reply README.md]: Thanks. I will fix it.

File: src/components/FastServices.tsx (hunk 4 above)
Comment (AndrewShmorhunGit) [reply src/components/FastServices.tsx:7]: Yes, it just stolen CSS code for time being.. I will sort it in next steps.

File: README.md (hunk 2 above)
Comment (AndrewShmorhunGit) [reply README.md]: Thanks. I will.

File: src/components/FastServices.tsx (hunk 4 above)
Comment (silvertipbear) [reply src/components/FastServices.tsx:7]: If this is not needed in this pr, then don't add this. Right now there is no option to test this css so I won't approve it.

**Review by AndrewShmorhunGit — COMMENTED** [review]
Hi Andrew. What do you think about that changes?

**Review by silvertipbear — APPROVED** [review]
Good job.

**Pull request merged**